setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = spacemake
-version = 0.7.2
+version = 0.7.3
```

Comment:
Now we have 0.7.4, so please change this accordingly.
When merging #64, this will be handled properly (automatically)

Reply:
Thx, just bumped version and will merge the PR